src/main/src/test/java/org/geoserver/catalog/CatalogIntegrationTest.java

```diff
@@ -42,6 +42,11 @@
 public class CatalogIntegrationTest extends GeoServerSystemTestSupport {
     
     @Override
+    protected boolean isFullInitRequired() {
```

Comment:
This test requires all beans to be initialized, as some programmatically attach catalog listeners, and this test is checking them. 99% of the test do not care/need this.